### PR TITLE
Git ignore templates

### PR DESCRIPTION
**Pull request by Jata2k**
Add sample gitignore files for new repositories

### CODE REVIEW

File: gitignore/go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Test output
+test.xml
+
+# Lint output
+lint.txt
+
+# Mac files
+.DS_Store
+._*
```
Comment (macreman) [inline gitignore/go]: we should include a section for ignoring build artifacts:

```
# Build artifacts
/{REPOSITORY_NAME}
{REPOSITORY_NAME}-*.zip
build.log
```

see https://github.com/companieshouse/chs-monitor-consumer/pull/15 for example in chs-monitor-consumer:

```
# Build artifacts
/chs-monitor-consumer
chs-monitor-consumer-*.zip
build.log
```

Comment (Jata2k) [reply gitignore/go]: Added

File: gitignore/ant [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Ivy libraries
+*/ivylib
+
+# Log files
+**/*.log
+**/*.log.*
+
+# Sonar directories
+.sonar
+**/.sonar
```
Comment (rafahop) [inline gitignore/ant]: Aren't these 2 equivalent?

Comment (Jata2k) [reply gitignore/ant]: Indeed they are

**Pull request merged**